### ISSUE TITLE
Issue 3: Construir el modelo CNN (Extractor de características y Clasificador)

### ISSUE THREAD

**Issue by sharafayyoub** [sharafayyoub/Actividad-redes-concolucionales#4]
Descripción:
En este paso crearás tu modelo CNN con Keras. Una CNN se divide en dos partes:

    -Extractor de características (capas convolucionales + max pooling).

     -Clasificador (capas densas).

El objetivo es que el modelo aprenda automáticamente los patrones visuales que permiten distinguir las clases del CIFAR-10.

-Tareas:

1. Crear un modelo secuencial:

from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Conv2D, MaxPooling2D, Flatten, Dense

model = Sequential()

2. Añadir el primer bloque convolucional:

Capa Conv2D con 32 filtros, kernel (3,3), activación 'relu' y input_shape=(32, 32, 3).

Capa MaxPooling2D(pool_size=(2,2)) para reducir la resolución y mantener solo las características relevantes.

3. Añadir el segundo bloque convolucional:

Capa Conv2D con 64 filtros y activación 'relu'.

Otra capa MaxPooling2D(pool_size=(2,2)).

4. Construir el clasificador:

Aplanar con Flatten().

Capa densa con 64 neuronas y activación 'relu'.

Capa de salida con 10 neuronas y activación 'softmax'.

5. Mostrar el resumen del modelo con model.summary() para verificar su estructura.

Criterio de éxito:
El modelo debe mostrar correctamente su arquitectura, con capas convolucionales seguidas de pooling y capas densas al final.
